docs/tasks/stateful-sets/debugging-a-statefulset.md

```diff
@@ -0,0 +1,78 @@
+---
```

Comment:
I'll add to index and tasks at the very end, right before merging, so as to avoid rebases.